doc/examples/segmentation/plot_rolling_ball.py

```diff
@@ -0,0 +1,270 @@
+"""
+================================================================
+Use rolling-ball algorithm for estimating background intensity
+================================================================
+
+The rolling-ball algorithm estimates the background intensity of a grayscale
+image in case of uneven exposure. It is frequently used in biomedical
+image processing and was first proposed by Stanley R. Sternberg in
+1983 [1]_.
+
+The algorithm works as a filter and is quite intuitive. We think of the image
+as a surface that has unit-sized blocks stacked on top of each other in place
+of each pixel. The number of blocks, and hence surface height, is determined
+by the intensity of the pixel. To get the intensity of the background at a
+desired (pixel) position, we imagine submerging a ball under the surface at the
+desired position. Once it is completely covered by the blocks, the height of
+the ball determines the intensity of the background at that position. We can
+then *roll* this ball around below the surface to get the background values for
+the entire image.
+
+Scikit-image implements this rolling-ball algorithm, as well as
+a generalized version which allows you to "roll" arbitrary ellipsoids. This
+generalized version is useful if you want to use different values for the
+spatial radius (``kernel_shape``) and the intensity amount
+(``intensity_vertex``).
+
+.. [1] Sternberg, Stanley R. "Biomedical image processing." Computer 1 (1983):
+    22-34. :DOI:`10.1109/MC.1983.1654163`
+
+
+Classic rolling ball
+-------------------------------
+
+In scikit-image, the rolling ball algorithm assumes that your background has
+low intensity (black), whereas the features have high intensity (white). If
+this is the case for your image, you can directly use the filter like:
+
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+import time
+
+from skimage import (
+    data, morphology, util, io
```

Comment:
```suggestion
    data, morphology, util
```
Can you please replace the `skimage.io` module by the `imageio` package? Deprecating `skimage.io` is in discussion, so we can start to give good habits in the documentation examples :wink:.